### PR TITLE
[INLONG-2410] Inlong Manager support business workflow suspend | restart | delete

### LINKED ISSUE #2410: [Improve] Inlong Manager support business workflow suspend | restart | delete
### Description

_No response_

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kipshi**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2410 

### Motivation

* Application group doesn't support suspend and restart

### Modifications

* use workflow engine to suspend | restart | delete application group resources

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/BusinessController.java [hunk 1]
```diff
@@ -86,7 +86,8 @@
     @OperationLog(operation = OperationType.DELETE)
     @ApiImplicitParam(name = "groupId", value = "Business group id", dataTypeClass = String.class, required = true)
     public Response<Boolean> delete(@PathVariable String groupId) {
-        return Response.success(businessService.delete(groupId, LoginUserUtil.getLoginUserDetail().getUserName()));
+        return Response.success(bizProcessOperation.deleteProcess(groupId,
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/BusinessController.java:89]: It is recommended to distinguish between deleting data and deleting processes.

Comment (kipshi) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/BusinessController.java:89]: BusinessProcessOperation will invoke BusinessService to logically delete data, I think data changing should all lay up cohensively in BusinessService, and repository class should not be invoked directly by other classes;

Comment (healchow) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/BusinessController.java:89]: OK, in `deleteProcess` method, logically delete business info.

**Review by healchow — APPROVED** [review]
LGTM.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**